code/modules/mob/living/carbon/human/human.dm

```diff
@@ -9,6 +9,7 @@
 	var/list/hud_list[9]
 	var/datum/species/species //Contains icon generation and language information, set during New().
 	var/embedded_flag	  //To check if we've need to roll for damage on movement while an item is imbedded in us.
+	var/percent_machine = 0
```

Comment:
Well I mean yeah just tune what you have pointed to this variable to the variable on the reagent